data/community_pages.yaml

```diff
@@ -324,3 +324,194 @@
     - form: ~
     - github: ~
   repository: ~
+- url: "ageofempires.fandom.com/wiki/Age_of_Empires_II:Portal"
+  https: true
+  description: "Everything you need to know from civs, to strategies to tech trees."
+  tags:
+    - "Reference"
+  preview: ""
+  contact:
+    - aoezone: ~
+    - email: ~
+    - twitter: ~
+    - discord: ~
+    - form: ~
+    - github: ~
+  repository: ~
+- url: "aokhotkeys.appspot.com"
+  https: true
+  description: "A hotkey editor with some popular presets"
+  tags:
+    - "Tools"
+  preview: ""
+  contact:
+    - aoezone: ~
+    - email: ~
+    - twitter: ~
+    - discord: ~
+    - form: ~
+    - github: ~
+  repository: ~
+- url: "aoecf.com"
+  https: true
+  description: "The Federation offers a motivation for players to engage with clans and a motivation for clans to be active and participate in clan events and tournaments"
+  tags:
+    - "Community"
+  preview: ""
+  contact:
+    - aoezone: ~
+    - email: ~
+    - twitter: ~
+    - discord: ~
+    - form: ~
+    - github: ~
+  repository: ~
+- url: "aoe2.guide"
+  https: true
+  description: "Tips, guides and build orders"
+  tags:
+    - "Guides"
+    - "Build orders"
+  preview: ""
+  contact:
+    - aoezone: ~
+    - email: ~
+    - twitter: ~
+    - discord: ~
+    - form: ~
+    - github: ~
+  repository: ~
+- url: "reddit.com/r/aoe2"
+  https: true
+  description: "r/aoe2: Community for AoE2 players"
+  tags:
+    - "Community"
+  preview: ""
+  contact:
+    - aoezone: ~
+    - email: ~
+    - twitter: ~
+    - discord: ~
+    - form: ~
+    - github: ~
+  repository: ~
+- url: "aoe2.se"
```

Comment:
Same as `siegeengineers.org`

Reply:
I think it wouldn't make sense to have something as `aliases`, just one page of them is enough? Thoughts?

Reply:
Removed the duplicate